R/MsBackendTimsTof-functions.R

```diff
@@ -77,9 +77,9 @@
   msg <- .valid_required_columns(x@indices, c("frame", "file"))
   if ("file" %in% colnames(x@indices) &&
       any(!x@indices[, "file"] %in% seq_len(length(x@fileNames))))
-    msg <- c(msg, "Some file indices are not valid")
+    msg <- c(msg, "Some file indices are not compatible with x@fileNames")
```

Comment:
As user I don't know what *not compatible with x@fileNames* means. What about *The following file indices are missing: ...*?

Reply:
Or simply "Some file indices are out of bounds"? It's rather that indices outside of the valid range of indices are used.

Reply:
I thought it would be a bit more informative that way but you are right for the user it's not clear. I would maybe change as Johannes suggested for the reason he mentioned

Reply:
or would you also print those that are out of bound?

Reply:
na. I would just say they are out of bound, not which ones are out of bound. This error should actually never happen if the user does not start to fiddle around in the internals of the object (which they are not supposed to do). So a plain "out of bounds" would be OK for me.